django/contrib/sites/checks.py

```diff
@@ -0,0 +1,11 @@
+from django.conf import settings
+from django.core.checks import Error
+
+
+def check_site_id(**kwargs):
+    if hasattr(settings, 'SITE_ID'):
+        if not isinstance(settings.SITE_ID, int):
```

Comment:
Can combine these two `if` statements with `and`.

```suggestion
    if hasattr(settings, 'SITE_ID') and not isinstance(settings.SITE_ID, int):
```